tests/integration.rs

```diff
@@ -5877,6 +5877,206 @@ mod parallel {
 
         handle_child_output(r, &output);
     }
+
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    fn test_vdpa_block() {
+        // Before trying to run the test, verify the vdpa_sim_blk module is correctly loaded.
+        if !exec_host_command_status("lsmod | grep vdpa_sim_blk").success() {
+            return;
```

Comment:
I was going to say it might be worth printing a message to say you're skipping it. But stderr is only shown on test failure. It's a shame `cargo test` lacks a skip ability.

Reply:
Yeah that's too bad :(